components/admin_console/__snapshots__/elasticsearch_settings.test.jsx.snap

```diff
@@ -13,7 +13,6 @@ exports[`components/ElasticSearchSettings should match snapshot, disabled 1`] =
       <FormattedMessage
         defaultMessage="Elasticsearch"
         id="admin.elasticsearch.title"
-        values={Object {}}
```

Comment:
Down with `defaultProps`! It's really annoying how changing it affects everywhere the component is even shallowly mounted